media/src/App.svelte

```diff
@@ -203,6 +196,84 @@
     let axesHolder = drawAxes({ gridMax, gridStep, axesMaterial });
     scene.add(axesHolder);
 
+    // Point elements
+    const tanFrame = new THREE.Object3D();
+    const arrowParams = {
+        radiusTop: gridStep / 10,
+        radiusBottom: gridStep / 20,
+        heightTop: gridStep / 7,
+    };
+    const arrows = {
+            u: new THREE.Mesh(
```

Comment:
I think moving these meshes up to the App component doesn't really makes sense - and keep in mind the pinpoint poll feature will only be used in a very particular context: when in a room and responding to a pinpoint poll. I imagine its functionality should largely be cordoned off to an independent class - i.e. something in the `polls/` directory.